katportalclient/test/test_client.py

```diff
@@ -29,64 +29,186 @@
+    },
+    {
+      "value": "91475",
+      "sample_time": "1523250002.0252408981",
+      "status" : "error"
+    },
+    {
+      "value": "91476",
+      "sample_time": "1523250012.0289709568",
+      "status" : "error"
+    },
+    {
+      "value": "91477",
+      "sample_time": "1523250022.0292000771",
+      "status" : "error"
+    }
+  ]
+}"""
+
+sensor_data2 = """{
+  "url": "/katstore/api/query/?start_time=1523249984&end_time=now&interval=0&sensor=sys_watchdogs_sys&minmax=false&buffer_only=false&additional_fields=false",
+  "sensor_name": "sys_watchdogs_sys",
+  "title": "Sensors Query",
+  "data": [
+    {
+      "value": "91475",
+      "sample_time": "1523250002.0252408981",
+      "status" : "error"
+    },
+    {
+      "value": "91476",
+      "sample_time": "1523250012.0289709568",
+      "status" : "error"
+    },
+    {
+      "value": "91477",
+      "sample_time": "1523250022.0292000771",
+      "status" : "error"
+    }
+  ]
+}"""
+
+sensor_data3 = """{
+  "url": "/katstore/api/query/?start_time=1523249984&end_time=now&interval=0&sensor=sys_watchdogs_sys&minmax=false&buffer_only=false&additional_fields=false",
+  "sensor_name": "sys_watchdogs_sys",
+  "title": "Sensors Query",
+  "data": [
+    {
+      "value": "91474",
+      "sample_time": "1523249992.0250809193",
+      "value_time": "1523249991.0250809193",
+      "status" : "error"
+    },
+    {
+      "value": "91475",
+      "sample_time": "1523250002.0252408981",
+      "value_time": "1523249991.0250809193",
+      "status" : "error"
+    },
+    {
+      "value": "91477",
+      "sample_time": "1523250022.0292000771",
+      "value_time": "1523249991.0250809193",
+      "status" : "error"
+    }
+  ]
+}"""
+
+sensor_data4 = """{
+  "url": "/katstore/api/query/?start_time=1523249984&end_time=now&interval=0&sensor=sys_watchdogs_sys&minmax=false&buffer_only=false&additional_fields=false",
+  "sensor_name": "sys_watchdogs_sys",
+  "title": "Sensors Query",
+  "data": [
+    {
+      "value": "91474",
+      "sample_time": "1523249992.0250809193",
+      "value_time": "1523249991.0250809193",
+      "status" : "error"
+    },
+    {
+      "value": "91475",
+      "sample_time": "1523250002.0252408981",
+      "value_time": "1523249991.0250809193",
+      "status" : "error"
+    },
+    {
+      "value": "91477",
+      "sample_time": "1523250022.0292000771",
+      "value_time": "1523249991.0250809193",
+      "status" : "error"
+    },
+    {
+      "value": "91477",
+      "sample_time": "1523250021.0292000771",
+      "value_time": "1523249990.0250809193",
+      "status" : "error"
+    }
+  ]
+}"""
+
+sensor_data_fail = """{
+  "url": "/katstore/api/query/?start_time=1523249984&end_time=now&interval=0&sensor=sys_watchdogs_sys&minmax=false&buffer_only=false&additional_fields=false",
+  "sensor_name": "sys_watchdogs_sys",
+  "title": "Sensors Query",
+  "data": []
+}"""
+
 # Example redis-pubsub message for sensor history
 sensor_history_pub_messages_json = {
```

Comment:
We aren't publishing the sensor history over the websocket anymore, so this can be removed.

The `mock_async_fetcher` doesn't need the `publish_raw_messages` and `client_states` anymore, so all that complication can be removed.  Same for `mock_async_fetchers`.